Storybook

```diff
@@ -12,16 +12,21 @@ RUN yarn install --immutable
 
 # Copy app files
 COPY . .
+
 # Build the app
 RUN yarn build-storybook
 
 # Bundle static assets with nginx
 FROM nginx:latest
+
 # Copy built assets from builder
-COPY --from=builder /app/dist/storybook/* /usr/share/nginx/html
+COPY --from=builder /app/dist/storybook/\* /usr/share/nginx/html
```

Comment:
Why escape `*` ?

Reply:
it's my prettier I don't know why, I remove it and just change the the node version